static/js/composebox_typeahead.js

```diff
@@ -343,10 +343,16 @@ export function tokenize_compose_str(s) {
 }
 
 export function broadcast_mentions() {
+    let wildcard_string = "";
+    if (compose_state.get_message_type() === "private") {
+        wildcard_string = "(Notify recipients)";
+    } else {
+        wildcard_string = "(Notify stream)";
+    }
```

Comment:
All strings must be tagged for translation, including these. See https://zulip.readthedocs.io/en/latest/translating/internationalization.html

I think you want the parens to be part of `special_item_text`, not this part, by the way; it'll be more ergonomic for translators.